src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1002,7 +1002,7 @@ ConnectivityManagerImpl::ConnectWiFiNetworkAsync(ByteSpan ssid, ByteSpan credent
     memcpy(keyStr, credentials.data(), credentials.size());
     g_variant_builder_add(&builder, "{sv}", "ssid", g_variant_new_string(ssidStr));
     g_variant_builder_add(&builder, "{sv}", "psk", g_variant_new_string(keyStr));
-    g_variant_builder_add(&builder, "{sv}", "key_mgmt", g_variant_new_string("WPA-PSK"));
+    g_variant_builder_add(&builder, "{sv}", "key_mgmt", g_variant_new_string("SAE WPA-PSK"));
```

Comment:
I think we should maybe comment here and below that using `wpa_cli get_capability key_mgmt` can be used to check capabilites and that we rely that SAE and WPA-PSK are available.

This may help people debug if this thing fails somehow.